test/unit/resources/etc_fstab_test.rb

```diff
@@ -0,0 +1,96 @@
+    _(entries.file_system_type).must_equal ['xfs', 'xfs', 'xfs', 'xfs', 'xfs', 'xfs', 'xfs', 'swap', 'iso9660', 'iso9660', 'tmpfs', 'nfs']
+    _(entries.mount_options).must_equal [['defaults', 'x-systemd.device-timeout=0'] , ['defaults', 'nodev', 'nosuid'], ['defaults', 'x-systemd.device-timeout=0', 'nodev', 'nosuid'], ['defaults', 'x-systemd.device-timeout=0', 'nodev', 'noexec', 'nosuid'],
+     ['defaults', 'x-systemd.device-timeout=0', 'nodev', 'nosuid'], ['defaults', 'x-systemd.device-timeout=0', 'nodev', 'nosuid'], ['defaults', 'x-systemd.device-timeout=0', 'nodev', 'nosuid', 'nodev', 'noexec', 'nosuid'], ['defaults', 'x-systemd.device-timeout=0'],
+     ['defaults', 'ro', 'noexec', 'noauto'], ['defaults', 'ro', 'noexec', 'noauto'], ['noexec', 'nosuid', 'nodev'], ['rsize=8192', 'wsize=8192', 'timeo=14', 'intr'] ]
+    _(entries.device_name).must_equal ['/dev/mapper/vg1-lv_root', 'UUID=ebffcd01-0695-4d4a-91a4-3a53c6f88d06', '/dev/mapper/vg1-lv_home', '/dev/mapper/vg1-lv_tmp', '/dev/mapper/vg1-lv_var', '/dev/mapper/vg1-lv_log', '/dev/mapper/vg1-lv_audit', '/dev/mapper/vg1-lv_swap',
+     '/dev/sr0', '/dev/cdrom', 'tmpfs', 'server:/usr/local/pub']
+    _(entries.file_system_options).must_equal [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+  end
+
+  it 'Verify etc_hosts filtering by `file_system_options`'  do
+    entries = resource.where { file_system_options == 0 }
+    _(entries.mount_point).must_equal ['/', '/boot', '/home', '/tmp', '/var', '/var/log', '/var/log/audit', 'swap', '/mnt/sr0', '/mnt/cdrom', '/dev/shm', '/pub']
+    _(entries.file_system_type).must_equal ['xfs', 'xfs', 'xfs', 'xfs', 'xfs', 'xfs', 'xfs', 'swap', 'iso9660', 'iso9660', 'tmpfs', 'nfs']
+    _(entries.mount_options).must_equal [['defaults', 'x-systemd.device-timeout=0'] , ['defaults', 'nodev', 'nosuid'], ['defaults', 'x-systemd.device-timeout=0', 'nodev', 'nosuid'], ['defaults', 'x-systemd.device-timeout=0', 'nodev', 'noexec', 'nosuid'],
+     ['defaults', 'x-systemd.device-timeout=0', 'nodev', 'nosuid'], ['defaults', 'x-systemd.device-timeout=0', 'nodev', 'nosuid'], ['defaults', 'x-systemd.device-timeout=0', 'nodev', 'nosuid', 'nodev', 'noexec', 'nosuid'], ['defaults', 'x-systemd.device-timeout=0'],
+     ['defaults', 'ro', 'noexec', 'noauto'], ['defaults', 'ro', 'noexec', 'noauto'], ['noexec', 'nosuid', 'nodev'], ['rsize=8192', 'wsize=8192', 'timeo=14', 'intr'] ]
+    _(entries.device_name).must_equal ['/dev/mapper/vg1-lv_root', 'UUID=ebffcd01-0695-4d4a-91a4-3a53c6f88d06', '/dev/mapper/vg1-lv_home', '/dev/mapper/vg1-lv_tmp', '/dev/mapper/vg1-lv_var', '/dev/mapper/vg1-lv_log', '/dev/mapper/vg1-lv_audit', '/dev/mapper/vg1-lv_swap',
+     '/dev/sr0', '/dev/cdrom', 'tmpfs', 'server:/usr/local/pub']
+    _(entries.dump_options).must_equal [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+  end
+
+  it 'verify etc_fstab can detect if a partition is mounted' do
```

Comment:
This isn't actually testing anything - it's creating a stub handler but it's never used.